nextflow_schema.json

```diff
@@ -10,22 +10,26 @@
             "type": "object",
             "fa_icon": "fas fa-terminal",
             "description": "Define where the pipeline should find input data and save output data.",
-            "required": [
-                "input"
```

Comment:
I don't think that this should be removed - input is required.

Reply:
It's really confusing but the input is defined a bit lower (if you search for "required" you'll find it). All of the parameters in the whole doc have been moved up and down... I could have done my first-commit more efficiently...

Reply:
Right, ok makes sense sorry - yes I had wondered which is why I gave up on my phone 😅